types/inboxsdk/index.d.ts

```diff
@@ -0,0 +1,1072 @@
+// Type definitions for InboxSDK 2.0
+// Project: https://www.inboxsdk.com/
+// Definitions by: Raphaël Doursenaud <https://github.com/rdoursenaud>
+//                 Amiram Korach <https://github.com/amiram>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+/*
+ * Copyright (c) 2016 GPC.solutions
```

Comment:
@amiram I believe you should add your own copyright ;)

Reply:
Sure. Do you want me to put your name as a maintainer? You'll get an email for approving each pull request to this code.

Reply:
I don't think I'm invested in InboxSDK enough but if you believe I can help, why not.